src/grid/tests/test_molgrid.py

```diff
@@ -349,11 +349,19 @@ def test_get_subgrid_1s(self):
         grid = MolGrid([atg1], BeckeWeights(), np.array([1]), store=False)
         fn = np.exp(-2 * np.linalg.norm(grid.points, axis=-1))
         assert np.allclose(grid.integrate(fn), np.pi)
+        # conventional subgrid
         subgrid = grid.get_subgrid(coords, 12.0)
-        fn = np.exp(-2 * np.linalg.norm(subgrid.points, axis=-1))
+        subfn = np.exp(-2 * np.linalg.norm(subgrid.points, axis=-1))
         assert subgrid.size < grid.size
         assert subgrid.size == 10560
-        assert np.allclose(subgrid.integrate(fn), np.pi)
+        assert np.allclose(subgrid.integrate(subfn), np.pi)
```

Comment:
Just leave a note for us. I think we agreed to use ``np.assert_allclose`` to replace redundant assert `np.alllcose`. So one point i will change all the assert command to compatible numpy.test command for better error message

Reply:
ok, I was indeed not sure anymore. I'll change the new ones now.

Reply:
Lol. It's too trivial anyway. There're some legacy tests with these old style. I think I am going to merge this PR. And maybe later we can reformat the tests into a more consistent style.